pkg/ui/workspaces/cluster-ui/src/databaseDetailsV2/tablesView.tsx

```diff
@@ -123,7 +123,10 @@ const COLUMNS: (TableColumnProps<TableRow> & {
     hideIfTenant: true,
     width: "fit-content",
     render: (t: TableRow) => (
-      <RegionNodesLabel nodesByRegion={t.nodesByRegion} />
+      <RegionNodesLabel
+        loading={t.nodesByRegion.isLoading}
+        nodesByRegion={t.nodesByRegion.data}
```

Comment:
why not pass `t.nodesByRegion` and let the component decide whether to do anything with `.isLoading`?